charts/testkube-cloud-api/templates/deployment.yaml

```diff
@@ -326,10 +326,16 @@ spec:
               value: "{{ .Values.api.features.bootstrapConfig.enabled}}"
             - name: "BOOTSTRAP_CONFIG_PATH"
               value: "/app/config/config"
+            - name: "TESTKUBE_DEFAULT_AGENT_TOKEN"
+              valueFrom:
+                secretKeyRef:
+                  name: testkube-default-agent-token
+                  key: agent-token
+                  optional: true
```

Comment:
Not sure if we can get away with always using this with `optional` or if we need to exclude this env when `bootstrapConfig.orgs.*.envs.*.agentToken.valueFrom.envRef` is used.

Reply:
Best to exclude unused environment variables.